_docs/reference/release-notes.md

```diff
@@ -88,5 +88,10 @@ identity provisioning. This agent runs on each node (VM / physical machine) and
 - **Persistent CA Key/Certificate Storage**. Istio CA now stores signing key/certificates in
 persistent storage to facilitate CA restarts. 
 
+### Known Issues
+
+- **User may get periodical 404 when accessing the application**:  We have noticed that envoy doesn't get routes properly occasionally thus a 404 is returned to the user.  We are actively working on this [issue](https://github.com/istio/istio/issues/1038).
+- **Pilot reports ready before Pilot is actually ready**: You can check pods status in the `istio-system` namespace and wait a few seconds after all Istio Pods reach ready status.  We are actively working on this [issue](https://github.com/istio/istio/pull/1055).
```

Comment:
pods status -> pod status
Istio Pods -> Istio pods
ready status -> ready state

Reply:
updated.  I kept status as that is what kubectl output shows when I call `kubectl get pods`